pkg/controller/storagecluster/reconcile.go

```diff
@@ -1190,6 +1190,11 @@ func newCleanupJob(sc *ocsv1.StorageCluster) *batchv1.Job {
 		"app": "ceph-toolbox-job-${FAILED_OSD_ID}",
 	}
 
+	const purgeOSDScript = `set -x
+if [[ \"$osd_status\" == \"up\" ]]; then echo \"OSD ${FAILED_OSD_ID} is up and running. Please check if you entered correct ID of failed osd!\"; else echo \"OSD ${FAILED_OSD_ID} is down. Proceeding to mark out and purge\"
```

Comment:
also add the command to get the osd_status for the passed osd_id

Reply:
ack